packages/sdk-js/src/GrowthBook.ts

```diff
@@ -1290,18 +1296,31 @@ export class GrowthBook<
   }
 
   public fireDeferredTrackingCalls() {
+    let hasInvalidTrackingCall = false;
     this._deferredTrackingCalls.forEach((call: TrackingData) => {
       if (!call || !call.experiment || !call.result) {
-        throw new Error("Invalid tracking data");
+        this.log("Invalid deferred tracking call", { call: call });
```

Comment:
Do a `console.error` directly here.  `this.log` is for debug statements only (`context.debug === true`), but I think this is something we want to make sure to surface to everyone in the browser console